src/main/webapp/javascript/user-page.js

```diff
@@ -61,13 +59,17 @@ async function loadTypeRecipes(type) {
   let recipesDivID = type + "-recipes";
   document.getElementById(recipesDivID).innerHTML = "";
   
+  // other will be used to set up the icon of the other add to list button
+  let other;
```

Comment:
Seems like you could get rid of this variable and following if statement, and move the logic below where you're calling changeIcons, like:
```
if (type == "planner") {
   changeIcons(recipesDiv, "cookbook")
} else {
   changeIcons(recipesDiv, "planner")
}
```
This way it's more flexible to future changes and you avoid storing an extra variable

Reply:
lol that makes a lot of sense